package.json

```diff
@@ -62,9 +73,14 @@
     "webpack-manifest-plugin": "1.3.2",
     "whatwg-fetch": "2.0.3"
   },
+  "peerDependencies": {
+    "patternfly-react": "^2.14.0",
+    "react": "^16.4.2",
+    "react-dom": "^16.4.2"
+  },
+  "dependencies": {},
   "scripts": {
-    "start": "node scripts/start.js",
-    "build": "node scripts/build.js",
+    "dist": "rm -rf dist; yarn lint && CI=nowatch yarn test && node scripts/dist.js",
```

Comment:
travis already does linting, maybe we should remove it as its included in this task?

Reply:
The point is, this target is called right before publishing the npm package. So final check to be sure ...

Reply:
that's fine, I am saying we could remove it in the travis configuration as it does it twice.

Reply:
Long-term, the whole project should be build-able (lint, validate, test, actual build) via single script, using `pre`-scripts to implement the logical chain of build steps.

> I am saying we could remove it in the travis configuration as it does it twice

+1 on removing `yarn lint` from `.travis.yml`, however the main point here is project-specific build steps leaking into Travis config. In other words, in the long-term, `.travis.yml` should only contain `yarn build` plus any CI-specific stuff like Coveralls publish.